consensus/src/genesis/genesis_init_config.rs

```diff
@@ -0,0 +1,19 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0
+
+lazy_static::lazy_static! {
+    pub static ref MAINNET_CONFIG: GenesisInitConfig = GenesisInitConfig::mainnet();
+    pub static ref DEVNET_CONFIG: GenesisInitConfig = GenesisInitConfig::devnet();
+}
+
+pub struct GenesisInitConfig {}
+
+impl GenesisInitConfig {
+    pub(crate) fn mainnet() -> Self {
```

Comment:
Do you think we can instead have two modules like:
```
consensus/config/mainet.rs
consensus/config/devnet.rs
```
`with #![cfg(feature = "<network>")]` in them? where `network = devnnet | mainnet`

Reason: I believe quite a few areas would need separate but similar functions for devnet and mainnet and that they would grow in future, so instead of having both APIs for devnet and mainnet together in the compiled library, we could just include either of them based on feature flag.

Also maybe in future we could extract config module to one level up so that it would contain config for other components as well, such as network configs and others.

Reply:
We can use conditional compilation in bin crate, making devnet | mainnet config/feature orthogonal in the lib crate might not be a good idea for some cases, e.g. testing, we will have to configure different feature set for the tests to cover all scenarios

Reply:
yeah, it make sense at the binary level.